src/components/Footer/footer.scss

```diff
@@ -1,5 +1,6 @@
 .footer-main {
   display: flex;
+  position: absolute;
```

Comment:
If you are using `position: absolute;`, please then use also some other params for positioning. ExampleL left, right and so